CONTRIBUTING.md

```diff
@@ -30,35 +32,38 @@ This section guides you through submitting a bug report. Following these guideli
 Explain the problem and include additional details to help maintainers reproduce the problem:
 
 * **Use a clear and descriptive title** for the issue to identify the problem.
-* **Describe the exact steps which reproduce the problem** in as many details as possible. 
+* **Describe the exact steps which reproduce the problem** in as many detail as possible. 
 * **Provide specific examples to demonstrate the steps**. 
 
 Provide more context by answering these questions:
 
+
 ### Suggesting Enhancements
 
 #### How Do I Submit A (Good) Enhancement Suggestion?
 
-
 * **Use a clear and descriptive title** for the issue to identify the suggestion.
 * **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
 * **Provide specific examples to demonstrate the steps**.
 * **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
 * **Explain why this enhancement would be useful** to most users.
 
+
 ### Your First Code Contribution
 
 Unsure where to begin contributing ? You can start by looking through these [Help wanted issues][help-wanted] issues.
 
-### Local development
+
+### Local Development
 
 #### Environment Variables
 
-You can edit .env to your liking (optional). This file should be .gitignored some time. 
+You can create and edit `.env.local` to your liking (optional).  This file will be ignored by git. 
```

Comment:
It means it isn't?
Shall we ignore it right now?

Reply:
Would you mind ignoring it in this PR then *bonvolu*? :blush:

Reply:
Well, the directives are already in `.gitignore`, I guess from a recipe of `symfony/framework-bundle`.

Reply:
Alright!